ui/src/app/pages/homepage/registration-dashboard.tsx

```diff
@@ -16,19 +16,19 @@ import {AccessModule, Profile} from 'generated/fetch';
 
 const styles = reactStyles({
   registrationPage: {
-    display: 'flex', flexDirection: 'column', paddingTop: '3%', paddingLeft: '3%',
+    display: 'flex', flexDirection: 'column',
     // Assign relative positioning so the spinner's absolute positioning anchors
     // it within the registration box.
     position: 'relative',
   },
   mainHeader: {
-    color: colors.primary, fontSize: 28, fontWeight: 400,
-    letterSpacing: 'normal', marginBottom: '0.2rem'
+    color: colors.primary, fontSize: '18px', fontWeight: 600,
```

Comment:
You should be able to use just `18` for font size, rather than `18px`

Reply:
I think I was just being consistent with the rest of the styles which use px.

Reply:
Just fewer characters. I don't have strong feelings. I think we have mixed styles on this, cause I've seen int rather than string elsewhere.